packages/sanes-chrome-extension/src/routes/share-identity/index.tsx

```diff
@@ -17,9 +17,9 @@ const ShareIdentity = ({ location }: RouteComponentProps): JSX.Element => {
 
   const req = requestContext.firstRequest;
   checkRequest(req, location, toast);
-  const { data, accept, reject } = req!; // eslint-disable-line
+  const { senderUrl, responseData, accept, reject } = req!; // eslint-disable-line
```

Comment:
I think it is better to disable specific rule for this line and not all rules like now.

Reply:
Refactored code to avoid the use of `!` entirely. Could you have another look?